src/Appwrite/Auth/Auth.php

```diff
@@ -29,7 +30,7 @@ class Auth
     ];
 
     public const DEFAULT_ALGO = 'argon2';
-    public const DEFAULT_ALGO_OPTIONS = ['memoryCost' => 2048, 'timeCost' => 4, 'threads' => 3];
+    public const DEFAULT_ALGO_OPTIONS = ['type' => 'argon2', 'memoryCost' => 2048, 'timeCost' => 4, 'threads' => 3];
```

Comment:
Was this an issue in the code base? another PR?

Reply:
This is to be able to resolve which GraphQL type the options represent when given only the options themselves, necessary for any union response type (currently only attributes and hashoptions): https://github.com/appwrite/appwrite/blob/feat-graphql-support/src/Appwrite/GraphQL/Types/Mapper.php#L354-L356